### DISCUSSION TITLE
Migration from Heroku to Google Cloud

### DISCUSSION THREAD

**Discussion by anischouchane** [NightscoutFoundation/xDrip#3226] [General]
Hello, 

I am a diabetic type one since 30 years and recently switched to iAPS, which is awesome. 

I set up my nightscout site recently with Heroku, but decided to switch to Google Cloud due to the cost. 

As instructed here (https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html), I set up my new nightscout page on Google Cloud successfully. Thank you by the way for the amazing guide!

Next, I migrated my Heroku site to Google Cloud as shown here: https://navid200.github.io/xDrip/docs/Nightscout/NS_Transfer.html

This brings me to my two questions: 
1. After the migration (transfer), I don't know how I can edit the variable of my nightscout. When I go to the Nightscout setup and try to edit variables using the text editor not all variables that I configured previously (on Heroku) appear. Can anyone please let me know how I can edit/remove these variables? 
2. I understand when using Google Cloud, MongoDB/Atlas is no longer needed. Can anyone give instructions on how to remove MongoDB from Nightscout? - especially considering my scenario where after the transfer I can't find the variables that I previously configured. 

Thank you in advance for the support!!

**Comment by Navid200** [comment]
The variables are not transferred by the transfer procedure.
You need to manually enter the variables you need in Google Cloud.

You can log into Heroku and view your variables.  You should be able to copy them one at a time and paste into Google Cloud.
There may also be a way to copy all of them at once into a single file.  But, I am not sure how.  

If you mean how to remove your database from Atlas, again, that is something that must be possible.  But, I don't know why you would want to do that considering you can keep the data there for free as long as you are not using it.  

@psonnera may be able to answer your questions about how to dump Nightscout variables and how to delete the database from Atlas.

**Comment by psonnera** [comment]
You can dump Heroku variables like this https://nightscout.github.io/troubleshoot/heroku/#backup-your-site-variables and I would just forget MongoDB Atlas: the database will fall asleep and that's it (you don't pay for Atlas, do you?).
You will need to update the Nightscout URL in iAPS.

**Comment by anischouchane** [comment]
Great and thank you both for that.

@Navid200, I already deleted the Heroku app so I can't copy them anymore but I will add the new variables manually (which I understand from the guide is better anyway). 

@psonnera, no I don't pay for Atlas, so I will just let it fall asleep and keep the data there for free. 

Cheers!
